mne/report/report.py

```diff
@@ -1739,7 +1739,10 @@ def _add_ica_artifact_sources(
     def _add_ica_artifact_scores(
         self, *, ica, scores, artifact_type, image_format, section, tags, replace
     ):
-        fig = ica.plot_scores(scores=scores, title=None, show=False)
+        assert artifact_type in ("EOG", "ECG")
+        fig = ica.plot_scores(
+            scores=scores, title=None, labels=artifact_type.lower(), show=False
+        )
```

Comment:
The CI failure on `old` looks related:
```
____________________________ test_manual_report_2d _____________________________
mne/report/tests/test_report.py:977: in test_manual_report_2d
    r.add_ica(
mne/report/report.py:1992: in add_ica
    self._add_ica(
mne/report/report.py:1880: in _add_ica
    self._add_ica_artifact_scores(
mne/report/report.py:1743: in _add_ica_artifact_scores
    fig = ica.plot_scores(
mne/preprocessing/ica.py:2596: in plot_scores
    return plot_ica_scores(
mne/viz/ica.py:1013: in plot_ica_scores
    raise ValueError(
E   ValueError: Need as many labels (0) as scores (1)
```
though I'm a bit confused by it

Reply:
Oh, that's unfortunate … I may have time to look into this tomorrow

Reply:
@larsoner This was a bug in the tests and should be fixed with [8948b64](https://github.com/mne-tools/mne-python/pull/12573/commits/8948b64b09dfead6726352d918ac0e26d7508697)